docs/_docs/framework/capabilities/manage/practice-ops.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+grand_parent: FinOps Framework
+parent: Manage
+title: FinOps Practice Operations
+permalink: /framework/capabilities/manage/practice-ops
+nav_order: zzz
+description: This article helps you understand the FinOps Practice Operations capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+---
+
+<!--
+author: bandersmsft
+ms.author: banders
+ms.date: 06/22/2023
+ms.topic: conceptual
+ms.service: finops
+ms.reviewer: micflan
+-->
+
+<span class="fs-9 d-block mb-4">FinOps Practice Operations</span>
+This article helps you understand the FinOps Practice Operations capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+  <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [▶️ Getting started](#️-getting-started)
+- [🏗️ Building on the basics](#️-building-on-the-basics)
+- [🍎 Learn more at the FinOps Foundation](#-learn-more-at-the-finops-foundation)
+- [⏩ Next steps](#-next-steps)
+- [ms.reviewer: micflan](#msreviewer-micflan)
+
+</details>
+
+---
+
+<a name="definition"></a>
+**The FinOps Practice and Operations capability focuses on building a robust FinOps team, defining clear responsibilities, and integrating FinOps practices into organizational processes to manage cloud costs effectively.**
+{: .fs-6 .fw-300 }
+
+Establish a dedicated FinOps team with the necessary skills, define and implement a FinOps strategy, foster stakeholder relationships, and develop a growth and maturity plan for the team.
+
+<br>
+
+## ▶️ Getting started
+
+To get started with this capability, follow these steps:
+
+- Establish clear roles and responsibilities within the FinOps practice. Initially, this may involve one or two individuals managing the FinOps responsibilities until a [dedicated team](docs\_docs\framework\capabilities\manage\structure.md) can be formed.
+- Define a clear FinOps strategy and integrate it with [organizational processes](docs\_docs\framework\capabilities\manage\intersecting-disciplines.md).
+- Build strong relationships with stakeholders and promote a [FinOps culture](docs\_docs\framework\capabilities\manage\culture.md) within the organization.
+- Develop a growth and maturity plan for the team's skills and capabilities.
+
+<br>
+
+## 🏗️ Building on the basics
+
+- Form a dedicated FinOps team with diverse skills and adequate resources.
+- Provide continuous [training and skill development](docs\_docs\framework\capabilities\manage\education.md) for team members.
+- Regularly review and refine the FinOps strategy to ensure alignment with organizational objectives.
+- Foster a culture of collaboration and communication across departments to enhance FinOps practices.
+
```

Comment:
Should we reference the FinOps Review assessment so the FinOps team can track progression? 

```suggestion
Leverage the [Microsoft FinOps Review](https://aka.ms/finops/review) to quantify progress.
```

Reply:
I added a reference to the assessment capability, which should cover us.